whois-api/src/test/java/net/ripe/db/whois/api/rdap/WhoisRdapServiceTestIntegration.java

```diff
@@ -315,6 +315,90 @@ public void lookup_inetnum_multiple_country_codes() {
         assertThat(ip.getName(), is("TEST-NET-NAME"));
         assertThat(ip.getLang(), is(nullValue()));
         assertThat(ip.getParentHandle(), is("IANA-BLK"));
+
+        final List<Notice> notices = ip.getNotices();
+        assertThat(notices, hasSize(4));
+        Collections.sort(notices);
+        assertThat(notices.get(0).getTitle(), is("Filtered"));
+        assertThat(notices.get(1).getTitle(), is("Multiple country attributes found"));
+        assertThat(notices.get(1).getDescription().get(0), is("There are multiple country attributes NL,DE in 192.0.0.0 - 192.255.255.255, but only the first country NL was returned."));
```

Comment:
Can I be a bit nitpicky and suggest we put spaces after a comma (NL, DE -> NL,DE)?